internal/dataplane/parser/translate_httproute.go

```diff
@@ -27,7 +33,17 @@ func ingressRulesFromHTTPRoutes(httpRouteList []*gatewayv1alpha2.HTTPRoute) (ing
 		}
 	}
 
-	return result, errs
+	if len(errs) > 0 {
+		for _, err := range errs {
+			// we log the error here instead of returning it for historical reasons
+			// and because this allows other HTTPRoute objects to be resolved. This
+			// loop structure was common at the time of writing but needs reworking.
+			// See: https://github.com/Kong/kubernetes-ingress-controller/issues/2130
```

Comment:
What was this? Not sure if this was just shifted from somewhere else, but this issue has been deleted.